### DISCUSSION TITLE
Make a different mod from lc-hax that only have the ability to control the entities

### DISCUSSION THREAD

**Discussion by arandomuser0** [winstxnhdw/lc-hax#473] [Ideas]
I know that lc-hax has that, but i want only that ability not others

**Comment by winstxnhdw** [comment]
You can just comment out the mods except for Possession and Phantom.

**Reply by arandomuser0** [reply]
yes these two possesion and phantom

**Reply by winstxnhdw** [reply]
Read the section on disabling mods.
